cmd/kubeadm/app/constants/constants.go

```diff
@@ -140,6 +140,12 @@ const (
 	// MasterConfigurationConfigMapKey specifies in what ConfigMap key the master configuration should be stored
 	MasterConfigurationConfigMapKey = "MasterConfiguration"
 
+	// KubeletBaseConfigurationConfigMap specifies in what ConfigMap in the kube-system namespace the init kubelet configuration should be stored
+	KubeletBaseConfigurationConfigMap = "kubelet-base-config-1.9"
+
+	// KubeletBaseConfigurationConfigMapKey specifies in what ConfigMap key the init kubelet configuration should be stored
+	KubeletBaseConfigurationConfigMapKey = "kubelet"
```

Comment:
FYI this will probably change to `kubelet.config.k8s.io` along with https://github.com/kubernetes/kubernetes/pull/53833